fms/modules/head.py

```diff
@@ -0,0 +1,18 @@
+import torch
+import torch.nn as nn
+
+
+class ClassHead(nn.Module):
```

Comment:
is this supposed to mean "classification"? if so we might want to spell it out, it looks like it means 'class' as in 'class ClassHead'

Reply:
I think I should create an `MLPHead` for this which is just MLP + the lm head as it is being used in multiple places.

Reply:
Switched this to only a ClassificationHead that can be reused for other headless models